pkg/client/group_mgmt_client.go

```diff
@@ -267,7 +268,7 @@ func (client *GroupMgmtClient) listPost(
 	params *param.GetParams,
 ) (interface{}, error) {
 	// build the url
-	url := fmt.Sprintf("%s/%s/detail", client.URL, path)
+	url := fmt.Sprintf("%s/%s/details", client.URL, path)
```

Comment:
When did this change?  And is this backwards compatible with older Nimble OS releases?

Reply:
Oh no . Its wrong . Let me fix this as part of PR#16. since this is already merged . sorry about that.